src/graphql/pullTranscendConfiguration.ts

```diff
@@ -116,6 +121,14 @@ export async function pullTranscendConfiguration(
               dataSubjects,
             )
           : undefined,
+      'prompt-a-vendor-email-settings': {
```

Comment:
we may want to only conditionally include these if the integration is type promptAPerson to avoiding cluttering the file for settings that don't do anything.